source/guide_fittrackee.rst

```diff
@@ -0,0 +1,186 @@
+ [isabell@stardust ~]$ cd ~/fittrackee
+ [isabell@stardust fittrackee]$ python3.9 -m venv fittrackee_venv
+ [isabell@stardust fittrackee]$ ls
+  fittrackee_venv
+ [isabell@stardust ~]$
+
+
+Activate the virtual environment and install Fittrackee:
+
+::
+
+ [isabell@stardust fittrackee]$ source fittrackee_venv/bin/activate
+  (fittrackee_venv) [isabell@stardust fittrackee]$ pip3.9 install fittrackee
+  (fittrackee_venv) [isabell@stardust fittrackee]$
+  ...
+  [notice] A new release of pip is available: 23.0.1 -> 23.2.1
+  [notice] To update, run: pip install --upgrade pip
+
+
+Then install version 1.26.6 of ``urllib3`` because otherwise you will get an error message later when setting up the database.
+
+::
+
+ (fittrackee_venv) [isabell@stardust fittrackee]$ pip3.9 install urllib3==1.26.6
+ ...
+ Successfully installed urllib3-1.26.6
+
+
+Configuration
+=============
+
+PostgreSQL
+----------
+
+Please follow the :manual:`PostgreSQL <guide_postgresql>` guide to configurate PostgreSQL.
+
+After that `create fittrackee database  <https://samr1.github.io/FitTrackee/en/installation.html#from-pypi>`_
+
+Web Backends
+------------
+
+With web backends you can connect Fittrackee running on port 5000 by default to the frontend to make it accessible from outside.
+Please follow the instructions :manual:`Web Backends <web-backends>`.
+
+
+Optional: Redis
+---------------
+
+We need Redis for task queue (if email sending is enabled and for data export requests) and API rate limits.
+
+Please follow the :lab:`Redis <guide_redis>` guide to setup redis with one variation: By default, Redis does not run on a port with us, but provides a Unix socket under ``/home/$USER/.redis/sock``. Either one has to adjust the configuration of the Fittrackee application or change the Redis configuration from `port 0' to 6379.
+
+Optional: Emails
+----------------
+
+In order to be able to send e-mails from Fittrackee, you need an e-mail account, which you specify in the next step in the file ``.env``. Any account can be used for this purpose. If you like, you can create the account directly at Uberspace.
+
+Please follow the :manual:`mail <mail-access>` guide to setup up Emails.
+
+
+Environment variables for Fittrackee
+------------------------------------
+
+You need to create the necessary environment variables for Fittrackee.
+You can use the `example file <https://github.com/SamR1/FitTrackee/blob/master/.env.example>`_ as a guide.
+Further explanations can be found in the `Fittrackee documentation <https://samr1.github.io/FitTrackee/en/installation.html#environment-variables>`_.
+
+Special note: For an installation in our system it is important to customize the variables ``HOST`` and ``DATABASE_URL`` as follows:
+
+::
+
+ export HOST=0.0.0.0
+ ...
+ export DATABASE_URL=postgresql://fittrackee:fittrackee@localhost:5432/fittrackee
+ ...
+
+For more information about the available ports, see :manual:`firewall ports <basics-ports>`.
```

Comment:
When I get your guide right you don't need to open any ports in the firewall, the database connection is local and your app can be reached with a web backend. Maybe just leave out this part? The manual article you link to is about opening ports in the firewall, which we don't need to to.